MAINTAINERS.yml

```diff
@@ -659,6 +660,7 @@ Release Notes:
     - jhedberg
     - fabiobaltieri
   files:
+    - doc/releases/migration-guide-*
```

Comment:
not sure if the new `doc/releases/eol_releases.rst` should be added as well?

Reply:
I like to use these mainly for the tag so it's easy to spot PRs that needs a fast turnaround to avoid merge conflicts so for that scope that file does not need to be there, changing that should just be a step in the release checklist and the PR does not need special treatment so I say let's let it fall under Documentation.